.circleci/config.yml

```diff
@@ -506,6 +548,13 @@ jobs:
           name: Install pnpm package manager
           command: |
             npm i pnpm --global
+      # A github dep clones-with-immutable-args is installed via github
+      # packages installed via npm via github automatically run postpack scripts
+      # their postpack script happens to use yarn so we need it here 
+      - run:
+          name: Install yarn package manager
+          command: |
+            npm i yarn@1 --global
```

Comment:
this yarn install and the pnpm install here and above can both be removed in a future pr after ci builder publishes